### PR TITLE
Fix minor formatting error in substring-transact-sql.md

### PR DESCRIPTION
**Pull request by cvionis**
Correct comma spacing in the "Syntax" section.

### CODE REVIEW

**Comment by Court72** [comment]
@MikeRayMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by cvionis** [comment]
#sign-off

File: docs/t-sql/functions/substring-transact-sql.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Returns part of a character, binary, text, or image expression in [!INCLUDE[ssNo
 ## Syntax  
   
 ```syntaxsql
-SUBSTRING ( expression ,start , length )  
+SUBSTRING ( expression, start , length )
```
Comment (MikeRayMSFT) [inline docs/t-sql/functions/substring-transact-sql.md]:
```suggestion
SUBSTRING ( expression, start, length )  
```

Why not let's make both of them match. 

Thank you for the recommendation.

**Comment by MikeRayMSFT** [comment]
#sign-off

**Review by MikeRayMSFT — APPROVED** [review]

**Pull request merged**